### PR TITLE
1928 onchain exact humanity score 90 day validity

### PR DESCRIPTION
**Pull request by aminah-io**
fixes: https://github.com/gitcoinco/passport/issues/1928
fixes: https://github.com/gitcoinco/passport/issues/1929

### CODE REVIEW

File: debug.md
Comment (lucianHymer) [inline debug.md:1]: Was this accidentally checked in?

File: docs/05-querying-passport-attestations-onchain.md [hunk 1]
```diff
@@ -63,4 +75,67 @@ This how the `providerVersions` is meant to be used:
...
+You can follow Steps 1 and Step 2 above to retrieve the attestation UID and score attestation.
+In order to retreive a passport attestation, there are 2 options:
+
+1. **Option A**: read from EAS and decode the attestation similar like for PAssport (see above)
+2. **Option B**: read from the 'cache' that has been implemented in the `GitcoinResolver` smart contract
+
+Given that the caching was introduced at a later point and not all scores written to EAS have been cached, we recommend the reading a users score:
+
+1. use **Option B** (fast and gas efficient)
+2. if **Option B** did not yield any result (`time` field is 0) then fallback to **Option A** as it is possible that the score you are looking for was not yet cached (it might have been issued before the cache has been introduced)
```
Comment (lucianHymer) [inline docs/05-querying-passport-attestations-onchain.md:95]: Is this accurate? The contract automatically falls back, right? Also, small type above in line 89 PAssport => Passport

File: docs/05-querying-passport-attestations-onchain.md [hunk 2]
```diff
@@ -63,4 +75,67 @@ This how the `providerVersions` is meant to be used:
...
+Score attestations are much simpler than Passport attestations. A Score attestation only contains 3 fields in its paylod:
+
+- uint256 score
+- uint32 scorer_id
+- uint8 score_decimals
+
+You can follow Steps 1 and Step 2 above to retrieve the attestation UID and score attestation.
+In order to retreive a passport attestation, there are 2 options:
+
+1. **Option A**: read from EAS and decode the attestation similar like for PAssport (see above)
```
Comment (lucianHymer) [inline docs/05-querying-passport-attestations-onchain.md:89]: Small typo PAssport

File: scripts/check.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
+// This script deals with deploying the GitcoinAttester on a given network
```
Comment (nutrina) [inline scripts/check.ts:1]: TODO document what this script is doing

File: yarn-error.log
Comment (nutrina) [inline yarn-error.log:1]: Remove this file

**Comment by nutrina** [comment]
- [ ] Updating the resolver needs to be done via multisig => transaction details need to be printed out to console.
- [ ] Update transfer ownership script to include Decoder SC

File: yarn-error.log
Comment (nutrina) [reply yarn-error.log:1]: Done

File: scripts/check.ts (hunk 3 above)
Comment (nutrina) [reply scripts/check.ts:1]: Done

**Pull request merged**